test/distributions/diten/blockCyclicDist.chpl

```diff
@@ -52,8 +52,8 @@ class BlockCyclicDom {
   type idxType;
   const whole: domain(nDims, idxType);
   const dist: unmanaged BlockCyclicDist(idxType);
-  const locDoms: [dist.localeDomain] unmanaged LocBlockCyclicDom(nDims, idxType);
-  proc initialize() {
+  var locDoms: [dist.localeDomain] unmanaged LocBlockCyclicDom(nDims, idxType);
```

Comment:
How about adding a comment like `locDoms is 'var' because it is set in postinit()` ?
Ditto `locArrs` below.

Reply:
That seems unnecessary to me, I think readers can easily determine where these fields are set.